stories/components/display/LoadingBody.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+import { Meta, Story } from "@storybook/react"
+
+import {
+  LoadingBody,
+  LoadingBodyProps,
+} from "../../../src/components/display/LoadingBody"
+
+const meta: Meta = {
+  title: "Components/Display/LoadingBody",
+  component: LoadingBody,
+  argTypes: {},
+  parameters: {
+    controls: { expanded: true },
+  },
+}
+
+export default meta
+
+const Template: Story<LoadingBodyProps> = (args) => <LoadingBody {...args} />
+
+export const Default = Template.bind({})
+export const CustomRowColumnCount = Template.bind({})
```

Comment:
I think it might be better if we just have a default story, the user can easily change the count props if they wish. same for both components